be/src/runtime/stream_load/stream_load_context.h

```diff
@@ -78,6 +78,12 @@ class KafkaLoadInfo {
 
 class MessageBodySink;
 
+const std::string TXN_BEGIN = "begin";
+const std::string TXN_COMMIT = "commit";
+const std::string TXN_ROLLBACK = "rollback";
```

Comment:
abort is a more general noun. Rollback is the following phase after abortion.

Reply:
`rollback` is standard SQL command used by both MySQL、PostgreSQL、Oracle， `abort` is only used by PostgreSQL  identical in behavior to `rollback` for historical reason